App.js

```diff
@@ -41,37 +41,30 @@ module.exports = function App() {
 
     return (
         <GestureHandlerRootView style={{ flex: 1 }}>
-            <KeyboardAvoidingView
```

Comment:
Could be missing something but is this line `const { KeyboardAvoidingView, SafeAreaView, Platform } = require('react-native');` good to cleanup?

Reply:
Good catch! I screwed up this merge a little bit 😅

Reply:
I do this all the time 😅